### ISSUE TITLE
help in making custom keras layer

### ISSUE THREAD

**Issue by gauravkr0071** [keras-team/keras#13831]
hey frnds, i am just a beginner , i want to make a dense layer in keras but with custom layer so i can work on more complex thing ?
how to make it ? just give me necessary code which i can easily put in between?
![git11](https://user-images.githubusercontent.com/51910127/75354631-51169680-58d3-11ea-95ed-15a5e0e8bcae.PNG)
import keras
from keras.layers import Input, Dense
from keras.models import Model
import numpy as np
import matplotlib.pyplot as plt
XOR_data=np.array([[0.1,0.1,1],   #--->1st set
                   [0.1,0.9,1],   #--->2nd set            ## used 0.1 instead of 0 for better learning
                   [0.9,0.1,1],   #--->3rd set            ## used 0.9 instead of 1 for better ---
                   [0.9,0.9,1]    #--->4th set
                  ]) 
XOR_data=XOR_data.reshape(4,1,3)
y_true=np.array([[0.1],      #--->y_true for 1st set
                 [0.9],      #--->y_true for 2nd set
                 [0.9],      #--->y_true for 3rd set
                 [0.1]       #--->y_true for 4th set
                ])  
y_true=y_true.reshape(4,1,1)

img_shape=(1,3)
input_vec = keras.Input(shape=img_shape)

output_1 = Dense(4, activation='sigmoid')(input_vec) 
output_2 = Dense(2, activation='sigmoid')(output_1 )# replace this one with custom layer 
y_pred = Dense(1, activation='sigmoid')(output_2)

model = Model(input_vec,y_pred)
model.summary()
model.compile(loss='mse',optimizer='adam',metrics=['accuracy'])

**Comment by sushreebarsa** [comment]
@gauravkr0071  Moving this issue to closed status as there has been no recent activity, in case you still face the error please create a new issue.Thanks!
